exercises/robot-name/robot-name-test.rkt

```diff
@@ -0,0 +1,38 @@
+#lang racket
+(require racket/class
+         "robot-name.rkt")
+
+(module+ test
+  (require rackunit
+           rackunit/text-ui)
+
+  (define proper-robot-name #px"\\p{Lu}{2}\\p{Nd}{4}")
```

Comment:
The regex should be:
`#px"\\p{Lu}{2}\\p{Nd}{3}"`

Reply:
Good catch! I also fixed the example.